stable/concourse/README.md

```diff
@@ -105,6 +105,7 @@ The following table lists the configurable parameters of the Concourse chart and
 | `web.nameOverride` | Override the Concourse Web components name | `nil` |
 | `web.replicas` | Number of Concourse Web replicas | `1` |
 | `web.resources` | Concourse Web resource requests and limits | `{requests: {cpu: "100m", memory: "128Mi"}}` |
+| `web.env` | Configure additional environment variables for the web container(s) | `[]` |
```

Comment:
Best practises are moving towards using `extraEnv` for this, to make it clear that they are added in addition to the already provided ones (if any).

Reply:
Great to know! Updating now :)

Reply:
Hmm, actually, as I think about this more, my concern is that `workers.env` already existed. So I'm worried about it being confusing that for worker pods you specify `env`, but for web pods you specify `extraEnv`.

So I guess the options are:
1. Use `web.extraEnv` and `worker.env` and just accept the discrepancy might be confusing.
2. Use `web.extraEnv` and switch `worker.env` to `worker.extraEnv`. My concern with that is that it would be a breaking API change, correct? I'm not really sure what best practice for that is in the helm chart community.
3. Use `web.env` and `worker.env` for consistency.

Let me know which option you think is best @timstoop !

Reply:
For me, option 2 looks best. You can signal a breaking change (although I think it's currently still ascetically only, helm users don't get a warning or anything) by bumping the major of the version. I see no reason not to do that, honestly. @frodenas @william-tran As chart owners, what do you think?

Reply:
Cool, that sounds good to me! I'll wait for signoff from @frodenas or @william-tran before updating the pr :)